streaming/text/convert/enwiki/mds/make_train_parallel.py

```diff
@@ -1,28 +1,68 @@
+# Copyright 2022 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Invoke create_pretraining_data.py on shard ranges on all the cores."""
+
 import os
+from argparse import ArgumentParser, Namespace
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Returns:
+        Namespace: Command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--in_pattern',
```

Comment:
Can you please add `help` for all the arguments to understand what each argument means? Thanks!

Reply:
Done